trace/trace_test.go

```diff
@@ -62,6 +63,159 @@ func TestIsValid(t *testing.T) {
 	}
 }
 
+func TestEqual(t *testing.T) {
+	a := SpanContext{
+		traceID: [16]byte{1},
+		spanID:  [8]byte{42},
+	}
+
+	b := SpanContext{
+		traceID: [16]byte{1},
+		spanID:  [8]byte{42},
+	}
+
+	if a.Equal(b) != true {
+		t.Error("Want: true, but have: false")
+	}
```

Comment:
Missing negative test where we verify equal returns false when it is expected.

Reply:
I'm not seeing where this was resolved. Can you point me in the right direction with a link to the added negative test?

Reply:
Oops, an oversight on my part.

Reply:
done